docs/releasing/publishing-from-a-support-branch.md

```diff
@@ -0,0 +1,162 @@
+# Publish GOV.UK Frontend when `main` branch has unreleasable changes
+
+Follow this document to publish a support branch of GOV.UK Frontend when the `main` branch contains unreleasable changes. For example, you might need to release a fix as part of a:
+
+- patch release, after the team has started to merge changes for a new feature release into the `main` branch - for example, a 3.14.x release once we've started merging changes for 3.15.0
+- release, after the team has started to merge changes for a new breaking release into the `main` branch - for example, a 3x.x release once we've started merging changes for 4.0.0
+
+Do not use this document to release changes for previous major versions. We have not tested the document in this scenario, and extra work would be needed to tell NPM not to mark the release as the 'latest'.
+
+If the `main` branch contains unreleased changes that are releasable immediately, [follow the steps in Publishing GOV.UK Frontend](/docs/releasing/publishing.md) instead.
+
+If the `main` branch only has a few unreleasable changes, you can temporarily revert these changes.
+
+1. Revert the unreleasable changes on the `main` branch.
+2. Publish GOV.UK Frontend.
+3. Add the reverted changes back into the `main` branch.
+
+However, this approach has risks. For example, it creates a messy commit history on the `main` branch.
+
+## Release GOV.UK Frontend from support branch
+
+### Before you release GOV.UK Frontend
+
+1. At stand up, tell the GOV.UK Design System team we are close to releasing.
```

Comment:
@vanitabarrett Also thought it would be good to specify what role is responsible for steps 1, 3 and 7 (the same way we've done this for steps 2, 4, 5 and 6). Is it always the Senior Frontend Developer who does these steps, or can it ever be any other role?

Reply:
Ah great, thanks @vanitabarrett!

```suggestion
1. At stand up, person leading the release (usually a developer) to tell the GOV.UK Design System team we are close to releasing.
```
FYI @jonathanglassman

Reply:
thank you